test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -763,10 +763,10 @@ void overwriteUncleHeaderForTest(mObject& uncleHeaderObj, TestBlock& uncle, std:
         return;
     }
 
-    string overwrite = "false";
```

Comment:
Is it necessary to have these testeth logic modifications in the PR introducing ConstantinopleFix?
It should better go to separate PR

Reply:
Some blockchain test filling was failing again. This as a part of the solution to refill the blockchain tests.